tests/unit/tf/horovod/test_embedding.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+
+import merlin.models.tf as mm
+from merlin.schema import ColumnSchema, Tags
+
+hvd = pytest.importorskip("horovod.tensorflow")
+sok = pytest.importorskip("sparse_operation_kit")
```

Comment:
Since horovod and sok are not currently installed in the CI, the unit test here will not run automatically for this test and we have to run the test manually for now. Can we make sure that the test will succeed in the latest container `nvcr.io/nvidia/merlin/merlin-tensorflow:22.11`? (or `22.12` which will be released shortly.)

Reply:
@marcromeyn